dataservice/api/common/schemas.py

```diff
@@ -14,18 +18,55 @@ class Meta:
         exclude = ('_id', 'uuid')
         dump_only = ('created_at', 'modified_at')
 
+    @pre_dump(pass_many=True)
+    def wrap_pre(self, data, many):
+        if isinstance(data, Pagination):
+            self.__pagination__ = data
+            if self.Meta.resource_url:
```

Comment:
I'm not seeing how/where self._links is used? Do you still need this bit of code?